src/controller.cpp

```diff
@@ -10,19 +10,18 @@ void Controller::configure(
   std::string name, const std::shared_ptr<tf2_ros::Buffer> & tf,
   const std::shared_ptr<nav2_costmap_2d::Costmap2DROS> & costmap_ros)
 {
-  auto node = parent.lock();
-
-
-  costmap_ros_ = costmap_ros;
+  parent_ = parent;
+  costmap_ros_ = std::move(costmap_ros);
```

Comment:
Don't move this, just assign that will mess up the reference counter, I suspect

Reply:
Oh.. i thought here it was passed by value. It won't have any effect here as far as i know. Removed